_test/test_multifit/test_multifit_horace_dnd_2.m

```diff
@@ -0,0 +1,90 @@
+classdef test_multifit_horace_dnd_2 < TestCaseWithSave
```

Comment:
can this be renamed more meaningfully?

Reply:
Undoubtedly. But this naming mirrors what is in the sqw equivalent .m files.

Replying to the observation that the multifit_*.m functions call mfclass with sqw as dataset class - the only effect that has (method: find "dataset_class" in all of Horace and Herbert) is that if the average property has been set for this multifit object, the using sqw will cause a more local average flag to be passed into the operation. Also noticing that while mfclass_horace_sqw_sqw does this with sqw, mfclass_Horace_sqw still does this with sqw_old. Guessing that the averaging may be a pixel data only process.